raiden_installer/installer_parts/funding.py

```diff
@@ -0,0 +1,173 @@
+import hashlib
+import requests
+import json
+from uuid import getnode
+from getpass import getuser
+from raiden_contracts.contract_manager import (
+    ContractManager,
+    contracts_precompiled_path,
+    get_contracts_deployment_info
+)
+from raiden_contracts.constants import (
+    CONTRACT_USER_DEPOSIT,
+    CONTRACT_CUSTOM_TOKEN
+)
+
+
+def goerli_funding(address: str) -> None:
+    '''
+    Fund address with Gö-ETH from faucet
+    '''
+    mac_address = getnode()
+    current_user = getuser()
+    client_hash = hashlib.sha256(
+        f'{mac_address}-{current_user}'.encode()
+    ).hexdigest()
+
+    try:
+        res = requests.post(
+            'https://faucet.workshop.raiden.network/',
+            json={'address': address, 'client_hash': client_hash}
+        )
+        print(res)
+    except requests.exceptions.RequestException as err:
+        print(err)
+
+
+class PfsAndMonitoringFunding:
+    '''
+    Creates a Web3 contract object for the custom token
+    contract and the user deposit contract and provides
+    methods for minting tokens, approving a deposit and
+    making the deposit.
+    '''
+    def __init__(self, w3, address, private_key):
+        contract_manager = ContractManager(contracts_precompiled_path())
+        user_deposit_abi = (
+            contract_manager.get_contract_abi(CONTRACT_USER_DEPOSIT)
+        )
+        custom_token_abi = (
+            contract_manager.get_contract_abi(CONTRACT_CUSTOM_TOKEN)
+        )
+
+        self.w3 = w3
+        self.address = address
+        self.private_key = private_key
+
+        self.chain_id = int(self.w3.net.version)
+
+        self.user_deposit_address = (
+            get_contracts_deployment_info(
+                self.chain_id
+            )['contracts'][CONTRACT_USER_DEPOSIT]['address']
+        )
+        self.user_deposit_contract = self.w3.eth.contract(
+            address=self.user_deposit_address,
+            abi=user_deposit_abi
+        )
+
+        custom_token_address = (
+            self.user_deposit_contract.functions.token().call()
+        )
+        self.custom_token_contract = self.w3.eth.contract(
+            address=custom_token_address,
+            abi=custom_token_abi
+        )
+        
+    def mint_tokens(
+        self,
+        token_amount: int,
+        gas: int,
+        gas_price: int
+    ) -> tuple:
+        mint = self.custom_token_contract.functions.mint(
```

Comment:
No need to concatenate all these functions to a single line - it does not cost company money to spread code out across several lines and variables.

```suggestion
        mint_result = self.custom_token_contract.functions.mint(token_amount)
        mint_result.buildTransaction(
            {
                'chainId': self.chain_id,
                'gas': gas,
                'gasPrice': gas_price,
                'nonce': self.w3.eth.getTransactionCount(self.address)
            }
        )
```